src/code-templates/services/order-service/package.json

```diff
@@ -38,6 +38,7 @@
     "sequelize-cli": "^6.4.1"
   },
   "devDependencies": {
+    "@jest-performance-reporter/core": "^2.1.2",
```

Comment:
Maybe we should use something that is more battle-tested?

Reply:
The other slow test reporters show the data in a very "easy to disregard" way.
No coloring and lines that look like they are not important.

After testing the other more popular reporters this seems to be the best fit for better testing experience.

The logic here is not so complex so I think it is fairly safe to use it.